### DISCUSSION TITLE
Charging curve for a charge session?

### DISCUSSION THREAD

**Discussion by MattiJarvinen** [tillsteinbach/VWsFriend#292] [Ideas]
It would be nice to have a charging curve in charging session like on the charging statistics.

Also could we have a Markdown file or something similar describing the current tables too. It would be a lot nicer to design new panels.

Maybe those could be added to the setting side of the application via postgre table describe or something?

**Comment by tillsteinbach** [comment]
For the first point: Which curve do you mean?

For the second point: I don't understand the idea. I think you have to go a bit more into details.
